src/main/resources/react/src/components/local/ua.js

```diff
@@ -16,12 +16,33 @@ const ua = {
 		logEmail: 'Введіть правильний логін або email',
 		logPassword: 'Неправильний пароль',
 		regEmail: 'Вкажіть діючу адресу електронної пошти',
-		regNick: 'Більше 2 і менше 50 символів, без пробілів',
+		regNick: 'Більше 1 і менше 50 символів, без пробілів',
```

Comment:
The same question as en.js has [here](https://github.com/Jack11M/EVO-Exchange-BE-2019/pull/175/files#r604223230)